cfme/tests/openstack/cloud/test_flavors.py

```diff
@@ -123,3 +123,11 @@ def cleanup():
 
     wait_for(lambda: not flavor.exists, delay=5, timeout=600, fail_func=flavor.refresh,
              message='Wait for flavor to appear')
+
+
+@pytest.mark.ignore_stream('5.9')
```

Comment:
Can you please comment why we are ignoring this for 5.9 ?

Reply:
because zero flavor disk is not supported in 5.9 yet